test/MathUtils.test.cpp

```diff
@@ -218,3 +218,31 @@ TEST(MathUtils, isRectInRectClass)
 	EXPECT_EQ(false, NAS2D::isRectInRect({5, 5, 1, 1}, {3, 3, 1, 1}));
 	EXPECT_EQ(false, NAS2D::isRectInRect({5, 5, 1, 1}, {7, 3, 1, 1}));
 }
+
+TEST(MathUtils, mapDomainToRange)
+{
+	//I know the temperature conversions have established formulas, but that's not what I'm testing.
+	//Given proper input I should get the same value as if I used the derived formulas.
+
+	//(v - minD) * (maxR - minR) / (maxD - minD) + minR;
+	//(F - 32.0f) * (5.0f / 9.0f) + 0.0f;
+	//(C - 0.0f) * (9.0f / 5.0f) + 32.0f;
```

Comment:
I think this comment could potentially be removed. If we choose two well known points from either scale, I don't think it should be necessary to demonstrate the formula. For example, the melting and boiling points of water: `0C..100C` => `32F..212F`

Btw, nice example for a test case.

Reply:
Good point. The formulas were a note for me as to what the input parameters to the function should be as they corresponded to the internal algorithm.

At the same time, it does demonstrate that you can't just  use `0,100,32,212` as input and get it to work...

Reply:
Disregard Previous. I just tested `EXPECT_NEAR(-17.22222f, NAS2D::convertDomainToRange(1.0f, 32.0f, 212.0f, 0.0f, 100.0f), 0.0001f);` (1F to C) using the fixed boiling/freezing points of water and it passed. Math is funny some times.